src/components/yeti/yetix.js

```diff
@@ -0,0 +1,117 @@
+import axios from 'axios';
```

Comment:
I may have missed something here? It's seems to me that you recoded some Vue reactivity mechanism? you could have the same result with a simple Vue component, and without any trick no ? 

```js
import Vue from 'vue';

export default new Vue({
    data() {
      return {
        activeTab: 0
      }
    },

    methods: {
      setActiveTab(index) {
        this.activeTab = index;
      }
    }
  });
```

Reply:
Well, I did not recoded Vue reactivity, but used `Vue.observable` (vue's own reactivity system). There is no real difference between what I used and your solution, except that I started with Vuex in mind so ended up with this code (separation between state, mutations and actions). One advantage is that each component can import only what needed, and you can easily find which components will mutate state.

Another point to have in mind is that Yeti now have a persistent state, and if a user go to `/yeti`, then in another page and come back to yeti, the state does not change. To keep things simple for now, I created a `defaultState`, copied it without reference (`JSON.parse(JSON.stringify(obj))`) to pass it to Vue reactivity system. The `setDefaultState` method reverts state back, every time YetiView is mounted. So the same kind of things should be done with your solution.

But I can go this way if you think it's simpler/better! Let me know.

Reply:
Sorry, I missed your answer...

> Another point to have in mind is that Yeti now have a persistent state

I may be wrong, but with the old-fashion solution, it's also like that: there is only one `activeTab` value in the entire WPA.

Well, to be honest, I'm definitly for a more solid solution using Vuex for the UI, **but** it would be a entire project. I don't feel that mixing both way of doing is the best solution, it would end with two design choice to understand for new contributors.

That being said, it's not a big deal, as it's used on a very particular part of the UI (I would have a stronger opinion for a plugin used everywhere :) ). So you have my opinion, I let you choose what you want, and I'll commit to your decision as if it would be mine :)